spec/datadog/tracing/transport/http_spec.rb

```diff
@@ -172,6 +172,7 @@
         Datadog::Core::Transport::Ext::HTTP::HEADER_META_LANG_VERSION => Datadog::Core::Environment::Ext::LANG_VERSION,
         Datadog::Core::Transport::Ext::HTTP::HEADER_META_LANG_INTERPRETER =>
           Datadog::Core::Environment::Ext::LANG_INTERPRETER,
+        'Datadog-Meta-Lang-Interpreter-Vendor' => RUBY_ENGINE,
```

Comment:
Minor: Was the hardcoding intentional here? In general I agree with having tests a bit less tied to the actual code (and thus prevent accidental edits/typos from automatically passing tests) but the inconsistency bothers me a bit 😅